src/homework6/n!.py

```diff
@@ -0,0 +1,48 @@
+"""Algorithm with O(n!) complexity: Travelling salesman problem"""
```

Comment:
It's too difficult task itself. Surely it has factorial solution (it's pretty obvious) and problem with this task is NP?=P problem. 
During our lectures I've meant regular tasks that are given as home tasks to students :) 🧪

Reply:
Well, you didn't mentioned that when you said about it, as far as I remember.
In this case you can just shuffle the list until it gets sorted)
I used sorting in my 5 homework

Reply:
> In this case you can just shuffle the list until it gets sorted) 

brilliant 🔥